src/components/common/editor/index.tsx

```diff
@@ -0,0 +1,57 @@
+import dynamic from "next/dynamic";
+import ReactQuill from "react-quill";
+import React from "react";
+const QuillNoSSRWrapper = dynamic(() => ReactQuill, { ssr: false });
```

Comment:
Not sure, why are we using Quill editor.

Reply:
This one is added by Rahul for text editing experiments, i guess